docs/pyproject.md

```diff
@@ -737,7 +737,17 @@ Here, we will have the `my_package_cli` script installed which will execute the
 When a script is added or updated, run `poetry install` to make them available in the project's virtualenv.
 {{% /note %}}
 
-### extras
+```toml
```

Comment:
Can you also update the `deprecation` notice above to include something like "This section should only be used for scripts of type 'file'".

Reply:
I've added that text.
But I don't actually understand what that means, has the `scripts` section been deprecated entirely?

Reply:
Might need to reword it if it is not clear I guess :) 

PEP 621 introduced both cli and gui scripts in the spec, but lacks the `type = "file"` functionality. So, the prefered way of doing cli/gui scripts is the `project.scripts` section.